stories/TextField/TextField.stories.js

```diff
@@ -3,15 +3,28 @@ import { storiesOf } from '@storybook/ember';
 import { action } from "@storybook/addon-actions";
 import textField from './textField.md';
 
-storiesOf("text field", module).add(
-  "default text field",
-  () => {
-    return {
-      template: hbs`{{bourbon-text-field placeholder="i am the placholder" onFocusOutOrEnter=onClick}}`,
-      context: { onClick: action("textFieldClick") }
-    };
-  },
-  {
-    notes: {markdown: textField}
-  }
-);
+storiesOf('text field', module)
+  .add(
+    'default text field with floating label',
+    () => {
+      return {
+        template: hbs`{{bourbon-text-field placeholder="i am the placholder" onFocusOutOrEnter=onClick}}`,
```

Comment:
Totally not customer-facing, but please update "placholder" to "placeholder" here and further down the file.